src/kustomer/webChat/client/src/kustomerServiceDesk.ts

```diff
@@ -154,18 +158,34 @@ class KustomerServiceDesk implements ServiceDesk {
   }
 
   private onMessageReceivedHandler = (response: OnMessageSentResponse, error: any) => {
+    if (error) {
```

Comment:
JSDoc should only be used when you're adding comments to things like functions. Comments for code should just use the normal `//` style of comments. All the comments you added except for on `ensureInit` should be changed.